libs/libglog/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=glog
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/google/glog/archive
+PKG_MD5SUM:=df92e05c9d02504fb96674bc776a41cb
+
+PKG_FIXUP:=autoreconf
+PKG_USE_MIPS16:=0
```

Comment:
I don't really remember. I have only tested ramips and it's working without it. I can remove it if you insist.